### PR TITLE
document awssns source

### PR DESCRIPTION
**Pull request by sameersbn**
ref: #49

### CODE REVIEW

File: docs/sources/awssns.md [hunk 1]
```diff
@@ -1,126 +1,105 @@
...
 
-Running the event source on your local machine can be convenient for development purposes.
+![Adding an AWS SNS source](../images/awssns-source/bridge-form-sns-source.png)
 
-### In the shell
+In the Source creation form, give a name to the event source and add the following information:
 
-Ensure the following environment variables are exported to your current shell's environment:
+* [**Secret**][accesskey]: Reference to a [TriggerMesh secret][tm-secret] containing an Access Key ID and a Secret Access Key to communicate with the AWS SNS API, as described in the previous sections.
+* [**AWS ARN**][arn]: ARN of the SNS topic, as described in the previous sections.
```
Comment (sebgoa) [inline docs/sources/awssns.md:80]: the screenshot also has an entry for `DeliveryPolicy`, should that be documented ?

File: docs/sources/awssns.md [hunk 2]
```diff
@@ -1,126 +1,105 @@
...
-  -e NAMESPACE=default \
-  -e K_LOGGING_CONFIG='' \
-  -e K_METRICS_CONFIG='{"domain":"triggermesh.io/sources", "component":"awssnssource", "configMap":{}}' \
-  gcr.io/triggermesh/awssnssource:latest
-```
+## Event Types
+
+The AWS SNS event source emits events of the following type:
+
+* `com.amazon.sns.notification`
```
Comment (sebgoa) [inline docs/sources/awssns.md:96]: then the actual SNS message is in the `data` section of the ce

File: docs/sources/awssns.md (hunk 1 above)
Comment (sameersbn) [reply docs/sources/awssns.md:80]: it comes from the bridge template

Comment (sameersbn) [reply docs/sources/awssns.md:80]: done!

File: docs/sources/awssns.md (hunk 2 above)
Comment (antoineco) [reply docs/sources/awssns.md:97]: I think we can assume knowledge about the CloudEvent spec in all docs?

File: docs/sources/awssns.md [hunk 3]
```diff
@@ -1,126 +1,107 @@
...
+                "sns:ConfirmSubscription"
+            ],
+            "Resource": "arn:aws:sns:*:123456789012:e2e-*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": "sns:Unsubscribe",
+            "Resource": "*"
+        }
```
Comment (antoineco) [inline docs/sources/awssns.md:64]: I think we should either rename or remove those `Sid` attributes. "VisualEditor" doesn't provide much value.

Comment (sameersbn) [reply docs/sources/awssns.md:64]: oh right.. fixing

Comment (sameersbn) [reply docs/sources/awssns.md:64]: updating rather...

Comment (sameersbn) [reply docs/sources/awssns.md]: please take another look

File: docs/sources/awssns.md (hunk 2 above)
Comment (sameersbn) [reply docs/sources/awssns.md:92]: yea i would also assume the reader to be familiar with the CloudEvent spec

**Comment by sameersbn** [comment]
@antoineco please take a look

**Comment by sameersbn** [comment]
ping!

**Review by antoineco — APPROVED** [review]

**Pull request merged**